<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_X">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>240</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Y">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>300</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Z">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>360</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_XX">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>240</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_YY">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>300</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_ZZ">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>360</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="clockwise">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>370</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>clockwise</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>440</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="counter">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>370</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>counter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="junk">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>junk</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
